servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Publisher.java

```diff
@@ -4062,6 +4062,13 @@ public static <T> Publisher<T> fromBlockingIterable(BlockingIterable<? extends T
      * Create a new {@link Publisher} that when subscribed will emit all data from the {@link InputStream} to the
      * {@link Subscriber} and then {@link Subscriber#onComplete()}.
      * <p>
+     * The resulting publisher is not replayable and supports only a single {@link Subscriber}.
```

Comment:
Great addition. Consider also clarifying what happens if the user happens to close the input stream as well.

Reply:
`java.io.Closeable#close()` method is idempotent per its javadoc, nothing will happen. If you mean something different, please provide an example of how you would phrase it.

Reply:
I was thinking of something like:

> If the `InputStream` is prematurely closed by the caller, no error will be raised.

I've been imagining a situation where the user might close the stream and no error is raised but the "payload" on the other side comes out shorter than expected.

Reply:
This is an implementation detail of a particular `InputStream`. There are some that ignore close, like `ByteArrayInputStream`, some return `-1` after closing, but there are some that throw `IOException` if they see a `read` attempt after `close`, like `PipedInputStream`.